src/mlpack/tests/range_search_test.cpp

```diff
@@ -1062,20 +1061,20 @@ BOOST_AUTO_TEST_CASE(EmptySearchTest)
 
   rs.Search(math::Range(0.0, 10.0), neighbors, distances);
 
-  BOOST_REQUIRE_EQUAL(neighbors.size(), 0);
-  BOOST_REQUIRE_EQUAL(distances.size(), 0);
+  REQUIRE(neighbors.size() == 0);
+  REQUIRE(distances.size() == 0);
 
   // Now check with a query set.
   arma::mat querySet = arma::randu<arma::mat>(3, 100);
 
-  BOOST_REQUIRE_THROW(rs.Search(querySet, math::Range(0.0, 10.0), neighbors,
+  REQUIRE_THROWS_AS(rs.Search(querySet, math::Range(0.0, 10.0), neighbors,
       distances), std::invalid_argument);
 }
 
 /**
  * Make sure things work right after Train() is called.
  */
-BOOST_AUTO_TEST_CASE(TrainTest)
+TEST_CASE("RangeTrainTest", "[RangeSearchTest]")
```

Comment:
This is changed because there was a conflict with `adaboost_test.cpp` at line : 740.